### DISCUSSION TITLE
Feature Request - Collection Dashboard Priorities

### DISCUSSION THREAD

**Discussion by Jegelewicz** [ArctosDB/arctos#4887] [General]
**Is your feature request related to a problem? Please describe.**
From #3983 

**Describe what you're trying to accomplish** 
Determine the most basic wish list for an institution/collection dashboard

**Describe the solution you'd like**
Pare down the list of dashboard wishes to something we can accomplish

**Describe alternatives you've considered**
We keep talking about it an nothing happens!

**Comment by Jegelewicz** [comment]
From today's Collection Dashboard Committee Meeting.

@dustymc @mkoo @lin-fred @jcabbott72 @cassandrabarber37 @acdoll @campmlc 

the priorities include:

## Publications and citations
  - Total number of publications citing records
  - Total by year (of publication)
## Taxonomic summaries 
 - orders
 - families
 - cultural divisions?
## Geographic coverage 
 - By country (maybe list top 10, then all else?)
 - % georeferenced
 - Map of collection localities
 - Map of use localities
 - Map of creation localities
## Collection Effort
 - Total number of records
 - Total by year (of collection)
 - Totals by month (of collection)
 ## Images/media
 - Total number of images
 - Total number of documents
 - Total by year (of image creation)

**Comment by Jegelewicz** [comment]
@dustymc can you provide the link to that pubs/citation thing you showed us in the meeting?

**Comment by dustymc** [comment]
https://arctos.database.museum/info/publicationbycollection.cfm?collection_id=1&citationonly=true&peerr=

**Comment by Jegelewicz** [comment]
During a [Symbiota webinar yesterday](https://www.youtube.com/watch?v=ipSIXOHud8A) I discovered that they add the GBIF citations widget to their collections - could we do this for ours?

https://www.gbif.org/article/1E6v02SFQyhupvB7JqDXPN/citation-widget

Also - I think adding the GBIF doi to any collections in Arctos that have one would make sense.

UTEP Plants:

![image](https://user-images.githubusercontent.com/5725767/182420267-70fc309a-b148-4f8d-84c9-726d7d316b83.png)

https://doi.org/10.15468/yhb6ky

**Comment by dustymc** [comment]
>could we do this

I hate everything about the GBIF "citation" model. We have a tremendous amount of work invested in making science [replicable](https://en.wikipedia.org/wiki/Replication_crisis) - the intersection of specimens and publications that we, and most everyone else, call "citations." Downloading huge swaths of data and even SORTA insinuating that somehow constitutes a 'citation' - and that is precisely what most, if not all, GBIF citations are - is evil.

Linking to the GBIF thingywidget doesn't get my blood pressure up quite so much, even if they refuse to reuse our identifiers. Adding "random GBIF page thingee identifier" to table collection and displaying it somewhere is actionable and could go back to Issues if there's some reason to do so. (I don't think I see one??)

**Reply by Jegelewicz** [reply]
Not saying I disagree - but presenting the number of citations at GBIF to a provost or NSF seems to get $ for digitization/better data?

**Reply by dustymc** [reply]
Yea that's probably a compelling argument, I'd still like to somehow attempt to avoid tainting the word 'citation'....

**Reply by Jegelewicz** [reply]
Good luck with that - I think the damage is done.

**Comment by dustymc** [comment]
Interesting idea. We say "Arctos has 378430 citations" (really), people (esp. admin types) hear 'smashed GBIF button maybe once' - yikes, maybe we need a better term for what we do?
